salt/transactional-update/init.sls

```diff
@@ -0,0 +1,31 @@
+/etc/systemd/system/transactional-update.service.d/10-update-rebootmgr-options.conf:
+  file.managed:
+    - makedirs: true
+    - source: salt://transactional-update/10-update-rebootmgr-options.conf
+    - user: root
+    - group: root
+    - mode: 644
+
+/etc/systemd/system/transactional-update.timer.d/10-increase-update-speed.conf:
+  file.managed:
+    - makedirs: true
+    - template: jinja
+    - source: salt://transactional-update/10-increase-update-speed.conf.jinja
+    - user: root
+    - group: root
+    - mode: 644
```

Comment:
For such small files, maybe it would be better to embed them here with `contents`...